test/nextjournal/clerk/analyzer_test.clj

```diff
@@ -227,7 +227,6 @@
 (str my-uuid)
 my-uuid"
                    analyze-string
-                   ana/analyze-doc
```

Comment:
analyze-string already calls analyze-doc. 

Now, maybe it's a bad idea to make analyze-doc not idempotent 🤔. It is so atm because I'm updating `:text` on blocks during analyze-doc to be stripped of clerk metas.

Reply:
Definitely need to have a new key!